packages/table/preview/index.tsx

```diff
@@ -186,28 +186,31 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
             return <Column
                 key={index}
                 renderColumnHeader={this.renderColumnHeader.bind(this)}
-                renderCell={this.renderCell.bind(this)}
+                renderCell={this.renderCell}
```

Comment:
Now declaring the function with a fat arrow to obviate the need for `.bind(this)`.